examples/async/src/lib.rs

```diff
@@ -18,16 +20,25 @@ pub unsafe extern "C" fn handle() {
 
 #[no_mangle]
 pub unsafe extern "C" fn handle_reply() {
-    gstd_async::block_on(handle_async());
+    gstd_async::resume();
```

Comment:
can we embed it in `gstd-async`? looks generic enough

Reply:
Do you mean combine `resume()` and `block_on()` into the one? `resume()` is already in `gstd-async`.

Reply:
The whole handle_reply thing

Reply:
Done.